index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+   <div class="Container">
+     <div class="Header">
+       <h3>Welcome</h3>
+       <img src="images/Enabled.png" alt="">
+     </div>
+   
+
+   <div class="headline">
+     <h2 class="welcoming">Hey there. I’m Anas
+        <span>I’m a software developer</span> 
+     </h2>
+     <p>
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+     </p>
+        <div class="contact">
+           <img src="images/github.png" alt="github">
+           <img src="images/medium.png" alt="medium">
+           <img src="images/twitter.svg" alt="twitter">
+       </div>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). You can insert your icon images into `li ` tags and wrap all of them into a `ul ` tag. Let's fix that